### ISSUE TITLE
[BUG] time date is not display correctly

### ISSUE THREAD

**Issue by JasonKhew96** [Unknow-Y/tojike-chrome-extension#3]
![image](https://user-images.githubusercontent.com/29531167/42422118-125cc2da-8313-11e8-9411-177e24b3549c.png)

**Comment by soyaine** [comment]
Please give more info about your browser and system.

**Comment by JasonKhew96** [comment]
```
Version 69.0.3496.0 (Official Build) canary (64-bit)
Windows 10 (1803)
```
![image](https://user-images.githubusercontent.com/29531167/42983261-51b5cb26-8c18-11e8-9ca8-685a5774fa05.png)

**Comment by JasonKhew96** [comment]
Resolved https://github.com/Unknow-Y/tojike-chrome-extension/commit/9796323ce8fcf96fb2858907d2925890e6d73f43

**Comment by soyaine** [comment]
Fix in version 2.0.9. Please try it again.
